volunteers_runbook.md

```diff
@@ -1,53 +1,70 @@
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+The Volunteer Organizer role is responsible for recruiting, organizing, and managing the volunteers for the event. 
 
 ### General Tasks
 
-Can you name typical tasks? For example:
+* Reach out to potential volunteers and request interest.
+* Maintain a list of awesome volunteers from years past to reach out to.
+* Provide feedback to the organizers on volunteer needs and opportunities
+* Create the two day volunteer schedule for the event.
+* Discuss volunteer needs for each functional with the associated organizer.
+* Communicate with other organizers and volunteers.
+* Ensure volunteers are running with assigned areas.
+* Google Drive folder for Volunteers: https://drive.google.com/drive/folders/1pcy2CV-V2OvecrZfWUmXGQwrqiEbHZdO
 
-* Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
 
 ## Important Dates
 
 What dates are important for this role? The following headings and details are examples.
 
 ### In the beginning
 
-* Come up with good stage jokes.
+* Create a new copy of the Volunteers folder in Google drive under this years event.
 
 ### 3 Months Out
 
-* Find vendors for your stage costume.
+* Review list of past volunteers and add any new volunteers you have had contact with.
+* Edit docs to match current year dates and links.
 
 ### 2 Months Out
 
-* Try on the costume.
+* Create a new Google Form for the new event for the year and update the text and information.
+* Make sure the volunteer answer sheet is setup and linked to the sign up form.
+* Send out the Call for Volunteers email to the list of folks who volunteered in prior years.
+* Make sure the coupon code for registration is setup. Reach out to Asif to get this done.
+* Set a deadline in the email of 2 weeks to sign up on the volunteer sheet.
 
 ### 1 Month Out
 
-* Sell out the sponsorships.
+* Chat with Volunteers in Slack to make sure they have everything setup.
+* Align with other Organizers and ensure each area has enough volunteers.
+* Create the event schedule sheet https://docs.google.com/spreadsheets/d/18cBLiLWDPy0xNHRIfQnghG6VTejMl3Q8/edit#gid=1156621535
+* Share the Volunteers Communication Plan with the group https://docs.google.com/document/d/15oYRf3rRCzBxJs7ZZ1Y-nwnbCYcW50SF/edit
+* Send out the load in doc for the volunteers https://docs.google.com/document/d/1Pp5Fb_QXI-jyyjq1h2oKpbXRwXEThbU0/edit
+* Send out the volunteers info doc https://docs.google.com/document/d/1hniuO72sleV68t6xxnA7yMihHlduTrdN/edit
 
 ### The Week Before
 
-* Panic.
+* Remind the volunteers of the schedule and who is assigned where.
+* Answer any questions folks have
 
 ### The Day Before
 
-* Set up.
+* Volunteers help with load in
 * Party.
 
 ### The Day Of
 
-* Get on stage.
+* Make sure all volunteers are where they need to be.
+* Walk the venue and check in on volunteers.
+* Monitor Slack to see if volunteers need any help.
 
```

Comment:
remember to add an "closing" section regarding load-out to storage and an "afterwards" section to schedule the volunteers lunch/dinner
maybe note something about proposing +1s for the vols social in the run-up to the conference, too, since we'll know what the budget situation is by 1 month out/1 week out... and
![family-dom](https://github.com/devopsdays-austin/runbooks/assets/2305030/6fd51ccf-5dee-4dd6-90d3-2d43c3fd6aa8)